### ISSUE TITLE
Making metadata field names more understandable

### ISSUE THREAD

**Issue by hebasta** [KorAP/Kalamar#40]
The metadata field names in the metadata view and the VC builder are for the user quite abundant and hard to understand, for example „pubDate“ or „corpusEditor“.

There could be different solutions for that:

- a link to a cheat table in the metadata view and the VC builder, which displays the metadata field names in a more understandable way

- descriptions to all metadata fields, that would be listed in the VC builder (see also descriptions in the annotation helper) and could be used for mouse overs in the metadata view

- User-settings: The user can define the metadata fields, he is interested in for meta data fetching. To see all meta data fields he clicks on a „more“ button. He can also choose a order of the metadata fields.

- More ideas are welcome ...

There are different point to consider by discussing the cheat table versus the descriptions of all metadata fields:
For both solutions localization has be kept in mind.
KorAP should be designed user-friendly and easy to use. The metadata field names and their abundance are for some users confusing. KorAP should not only be designed for the expert user, but serve the needs for different kind of users. Using mouse overs in the metadata view is laborious.

On the other hand descriptions can be as laborious as necessary. Users using the cheat sheet couldn't talk about metadata with the other users, as they would talk about different names. Specific metadata names may not only be used by annotators, but also by other corpus analysis tools - and these names are used project wide. KorAP has also users who are organized in groups that compile their own corpora. It is therefore an straightforward solution to display the internal field names.

**Comment by kupietz** [comment]
I'd opt for (first) mouse-overs and (then) a page with a collection of all descriptions – ideally based on the same data source. (Keep the corpus data itself and maybe also the TEI-specification as additional or fallback data sources in mind.)

IMHO, a user-selectable display would currently be either overkill for the user (too many options) or for us to implement. As a possible approach, I could imagine something like the overview display in e-mail clients where you can often drag, show and hide the fields in the table header. But I don't know of any JavaScript-library that would support that currently and compared to other missing functions it's not that important, or is it?

**Comment by hebasta** [comment]
> (then) a page with a collection of all descriptions
Do you mean a documentation page with all the descriptions?

> IMHO, a user-selectable display would currently be either overkill for the user (too many options) or for us to implement.
I added it for the sake of completeness because I understood we implement something like that in the distant future. Should I close the issue because it is not much needed at the present time?

**Comment by kupietz** [comment]
> Do you mean a documentation page with all the descriptions?

Yes, but that could be identical to the "cheat sheet" option and we should still discuss this.

One more thing: For some metadata categories, the range of possible values will be more important and helpful than just a description.

> I added it for the sake of completeness because I understood we implement something like that in the distant future. Should I close the issue because it is not much needed at the present time?

No, "for the sake of completeness" and "distant future" sounds good :)

**Comment by Akron** [comment]
As discussed before, an option to limit the shown metadata categories (with a "more meta" button) may also be helpful, so the user only sees metadata categories he is really interested in.

**Comment by kupietz** [comment]
While we don't have  KorAP-functionalities to show possible metadata values, we should maybe provide them with the corpus description on DeReKo releases.
